library/alloc/src/sync.rs

```diff
@@ -1327,12 +1790,12 @@ impl<T: ?Sized> Clone for Arc<T> {
             abort();
         }
 
-        Self::from_inner(self.ptr)
+        Self::from_inner_in(self.ptr, self.alloc.clone())
     }
 }
 
 #[stable(feature = "rust1", since = "1.0.0")]
-impl<T: ?Sized> Deref for Arc<T> {
+impl<T: ?Sized, A: Allocator> Deref for Arc<T, A> {
```

Comment:
Woah, I'm editing code that hasn't been touched in 7 years

Reply:
Heh, I think this applies to many Deref methods. 😄